src/Panels/Panels.js

```diff
@@ -1049,7 +1049,9 @@ module.exports = kind(
 		}
 		var orig = ev.originator,
 			idx = this.getPanelIndex(orig),
-			target = this.getSpotlightTarget('LEFT', orig);
+			target = this.getSpotlightTarget('LEFT', orig),
+			active = this.getActive(),
+			h = active ? active.getHeader() : null;
```

Comment:
I thought that there are two separate cases.
- no active panel (No panel pushed) 
- no header (Custom panel case)